com.unity.ml-agents/Runtime/SideChannels/TrainingAnalyticsSideChannel.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using UnityEngine;
+using Unity.MLAgents.Analytics;
+using Unity.MLAgents.CommunicatorObjects;
+
+namespace Unity.MLAgents.SideChannels
+{
+    public class TrainingAnalyticsSideChannel : SideChannel
+    {
+        const string k_TrainingAnalyticsConfigId = "b664a4a9-d86f-5a5f-95cb-e8353a7e8356";
```

Comment:
was this generated by you, or someone on the analytics team?  just curious.

Reply:
nevermind

Reply:
Yeah, it's a copy-paste of the python side. If you know of a way to consistently generate GUIDs on both languages from a seed, I'm all ears. In the meantime, I've been using `uuid.uuid5(uuid.NAMESPACE_URL, "com.unity.ml-agents/[SideChannelName]")` on python and pasting into C#.